README.md

```diff
@@ -21,6 +21,23 @@ This action for [Changesets](https://github.com/atlassian/changesets) creates a
 
 ### Example workflow:
 
+> [!WARNING]
+>
+> Before run one of the following workflows be sure that the `GITHUB_TOKEN` has the permission to access the repository contents and create a PR
+> You can do this:
+> - [Using repository Action settings](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-github-actions-settings-for-a-repository#setting-the-permissions-of-the-github_token-for-your-repository)
+> 
+>   Under "Workflow permissions" section
+>   - choose "Read and write permissions"
+>   - enable "Allow GitHub Actions to create and approve pull requests"
+>   
+> - [Using `permissions` key in the workflow file](https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#permissions)
```

Comment:
This is the preferred way to do this, I'd put it first

Reply:
Hmm, checking `Allow GitHub Actions to create and approve pull requests` seems to be mandatory in either case

Reply:
I had an old repository where I used the permissions key and never tackled with settings.
After re-checking it, I see that `Allow GitHub Actions to create and approve pull requests`  is checked so maybe it was the default value sometimes ago and then it has been changed.

I'll update the content. Thanks for reporting this!